onnx2pytorch/convert/layer.py

```diff
@@ -22,9 +31,9 @@ def extract_params(params):
 
 def load_params(layer, weight, bias):
     """Load weight and bias to a given layer from onnx format."""
-    layer.weight.data = torch.from_numpy(numpy_helper.to_array(weight))
+    layer.weight.data = _deserialize_to_torch(weight)
```

Comment:
Yeah, so this is helpful is because of stuff we do when training the converted model. I can try condense it down into a simple test case, though, if you'd like.